tests/tls/tls-store-02/suricata.yaml

```diff
@@ -7,11 +7,6 @@ outputs:
       types:
         - alert
         - tls
-  - tls-log:
-      enabled: yes  # Log TLS connections.
-      filename: tls.log # File to store TLS logs.
-      append: yes
-      extended: yes     # Log extended information like fingerprint
```

Comment:
perhaps these tests should be duplicated with an lt-version else it remains untested on 7 too